frontend/src/Home.css

```diff
@@ -24,12 +24,15 @@
         text-align: center;
         font-family:'Courier New', Courier, monospace; }
 .Rest {
-  border: 3px solid rgb(0, 0, 0);
   background-color: rgb(236, 98, 17);
-  margin: 0 25em 1em 25em;
+  margin: 0 20em 1em 20em;
   text-align: center;
   font-family:'Courier New', Courier, monospace; 
 }
+.Image {
+  margin: auto;
+  width: 65%;
```

Comment:
Can we scale this to a consistent size so variable resolution images appear the same?

Reply:
This was addressed in subsequent commits.